linkup-cli/src/remote_local.rs

```diff
@@ -7,49 +7,58 @@ use crate::{
     CliError, LINKUP_LOCALSERVER_PORT,
 };
 
-pub fn remote(service_name: String) -> Result<(), CliError> {
+pub fn remote(service_names: Vec<String>) -> Result<(), CliError> {
+    if service_names.is_empty() {
+        return Err(CliError::NoSuchService(
+            "No service names provided".to_string(),
+        ));
+    }
     let mut state = get_state()?;
 
-    let service = state
-        .services
-        .iter_mut()
-        .find(|s| s.name == service_name)
-        .ok_or(CliError::BadConfig(format!(
-            "Service {} not found",
-            service_name
-        )))?;
-    service.current = ServiceTarget::Remote;
+    for service_name in service_names.clone() {
+        let service = state
+            .services
+            .iter_mut()
+            .find(|s| s.name == service_name)
+            .ok_or(CliError::NoSuchService(service_name))?;
+        service.current = ServiceTarget::Remote;
+    }
```

Comment:
**_suggestion/nit:_**
You can avoid having to clone the whole array of services names if you use a reference here. It should be enough to clone just in case of a failure.
Something like:
```suggestion
    for service_name in &service_names {
        let service = state
            .services
            .iter_mut()
            .find(|s| &s.name == service_name)
            .ok_or_else(|| CliError::NoSuchService(service_name.into()))?;
        service.current = ServiceTarget::Remote;
    }
```

Note: The change of `.ok_or_else` instead of `.ok_or` is also to avoid extra allocation. More info here: https://stackoverflow.com/a/45547401/4043683